doc/palisade_clients.md

```diff
@@ -21,23 +66,13 @@ spark.readStream
 We would expect that Spark would create a filter to be pushed down to the data reader that says it only wants "colA" and "colB".
 
 The Spark client for Palisade would create a user (based on who is running the command) and take the resource, purpose and filter to register a data request with the Palisade service.
-Then the Spark client would then take the response of that request and split the list of resources over the number of executors Spark is running and get each executor to request access to the subset of resources assigned to that executor.
+Then the Spark client would then take the response of that request and split the list of resources over the number of executors Spark is running.
+Then it would get each executor to request access to the subset of resources assigned to that executor.
 
 The data service receives the request for data and passes that request to the palisade service to validate the request and to get the data access policies that apply to those resources.
 Then the data service can read the resource applying the column selection (columns requested plus any columns required to apply the policy) then apply the policies, transform the data so only the two requested columns are returned and then stream the data back to the executor that made the request.
```

Comment:
```suggestion
The Data Service receives the request for data and passes that request to the palisade service to validate the request and to get the data access policies that apply to those resources.
Then the Data Service can read the resource applying the column selection (columns requested plus any columns required to apply the policy) then apply the policies, transform the data so only the two requested columns are returned and then stream the data back to the executor that made the request.
```
I would also change the wording about how the request is validated. This is no longer done by passing it into the Palisade Service but checking redis to see if the AMS stored a record.